docs/arcaflow/getting-started.md

```diff
@@ -1,101 +1,97 @@
 # Arcaflow Getting Started Guide
 
-## Step 1: Get a container engine
+## Running Workflows
+An Arcaflow workflow is a definition of steps structured together to perform complex actions. Workflows are defined as machine-readable YAML and therefore can be version-controlled and shared easily to run in different environments. A workflow is a way of encapsulating and sharing expertise and ensuring reproducible results.
 
-In order to use Arcaflow, you will need a container engine on your computer. For the purposes of this guide, we'll assume you are using Docker or Podman.
+Running a workflow only requires having the [Arcaflow engine binary](https://github.com/arcalot/arcaflow-engine/releases), the workflow definition file, and in most cases an input file. A config file is also optional and in some cases needed, depending on the target of the workflow. All of these files are typically defined in YAML. Additionally, the target of the workflow needs a compatible container platform, such as Podman, Docker, or Kubernetes.
 
-## Step 2: Get the engine
+!!! note
+    The default container platform for the Arcaflow engine is Podman. To use another platform, a [configuration file](/arcaflow/running/setup/#configuration) is required.
 
-Head on over to the [GitHub releases page](https://github.com/arcalot/arcaflow-engine/releases) and download the latest release.
+A repository of [example workflows](https://github.com/arcalot/arcaflow-workflows) is available for reference and practice. Let's try running the [basic example](https://github.com/arcalot/arcaflow-workflows/basic-examples/basic).
 
-## Step 3: Create your first plugin
+First we will clone the example workflows repository and set our working directory:
 
-Let's create a simple hello-world plugin in Python. We'll publish the code here, you can find the details in the [Python plugin guide](plugins/python/first.md).
-
-```python title="plugin.py"
-#!/usr/local/bin/python3
-import dataclasses
-import sys
-from arcaflow_plugin_sdk import plugin
-
-
-@dataclasses.dataclass
-class InputParams:
-    name: str
-    
-    
-@dataclasses.dataclass
-class SuccessOutput:
-    message: str
+```bash
+git clone https://github.com/arcalot/arcaflow-workflows.git
+export WFDIR=arcaflow-workflows/basic-examples/basic/
+```
 
+Then we will run the workflow, setting the working directory as the context, and defining the workflow, configuration, and input files to use:
 
-@plugin.step(
-    id="hello-world",
-    name="Hello world!",
-    description="Says hello :)",
-    outputs={"success": SuccessOutput},
-)
-def hello_world(params: InputParams):
-    return "success", SuccessOutput(f"Hello, {params.name}")
+```bash
+arcaflow --context $WFDIR --workflow workflow.yaml --config config.yaml --input input.yaml
+```
 
+Arcaflow will display logs, depending upon the configured verbosity, and then will return the machine-readable output of the workflow in YAML format:
```

Comment:
"machine-readable output [...] in YAML" here seems rather unhelpful. YAML *is* machine-readable.

Reply:
I read this as "machine-readable output, specifically in YAML"

Reply:
Why is our *output* YAML explicitly "machine-readable" but not our *input YAML*, or *workflow YAML*, or *config YAML*?

I mean, yeah, it's all by definition machine-readable -- adding that explicitly here and only here seems silly. (And adding "machine-readable" before every file we expect to in fact be machine-readable seems even more silly.)

Reply:
It's important here because the workflow output is the final result of Arcaflow's job, and the point is to inform the user in a succinct way that that output can then be passed to another "system" with ease.

Reply:
If you're assuming a reader doesn't know that YAML is "machine readable", we shouldn't be assuming they know what YAML means, or "file". In this context, the phrase just feels silly. Oh, well.